RedHatInsights/compliance-frontend
# fix(EditPolicy): RHICOMPL-3953 - Properly request rule tree

How to test:

1) Open a policy (with and without systems) in Compliance to edit
3) Go to the systems tab, add (new) systems to the policy
3) Go to the rule tab and verify the rule tree is loaded
4) Close the modal and open it again and verify again that the rules are in tree form.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
